api/taxonomies/views.py

```diff
@@ -0,0 +1,74 @@
+from rest_framework import generics
+
+from api.taxonomies.serializers import TaxonomySerializer
+from api.base.views import JSONAPIBaseView
+
+import json
+
+class PlosTaxonomyMixin(JSONAPIBaseView, generics.RetrieveAPIView):
```

Comment:
The `PlosTaxonomyMixin` class should just inherit from `(object)` and make the two individual views below to inherit from `JSONAPIBaseView`, `generics.RetrieveAPIView` as well as `PlosTaxonomyMixin` individually -- more repetitive I know, but is how mixins generally work